pkg/config/config.go

```diff
@@ -340,7 +344,7 @@ func (m *Manager) setDefaults(prefix string, flags *pflag.FlagSet, config interf
 		fileOnly := field.Tag.Get("file-only")
 
 		if configValue.Field(i).CanInterface() {
-			fieldType := field.Type.Kind()
+			fieldKind := field.Type.Kind()
```

Comment:
This is extremely confusing - the value represents `reflect.Kind`, not `reflect.Type`, hence `fieldType` is absolutely incorrect name for this variable